package.json

```diff
@@ -8,12 +8,12 @@
   },
   "license": "MIT",
   "main": "./dist/index.js",
+  "types": "./typings.d.ts",
   "scripts": {
     "start": "start-storybook -p 6006",
-    "watch": "babel src -w -d dist --ignore src/stories",
-    "build": "babel src -d dist --ignore src/stories",
+    "build": "rimraf dist && tsc",
     "build:storybook": "build-storybook -o .out",
-    "test": "npm run lint && mocha --compilers js:babel-core/register test/*.spec.js",
+    "test": "npm run lint && mocha --require ts-node/register test/*.spec.js",
     "lint": "eslint src test",
     "precommit": "lint-staged",
     "prettify": "prettier-eslint --write \"src/**/*.js\" \"test/**/*.js\"",
```

Comment:
I guess that we should now also target `ts`, `tsx` & `jsx`?

Reply:
And probably (later?) also add tslint along with the configuration from the UI?

Reply:
Yeah eventually that would be good to include tslint.

Reply:
This won't target ts or tsx as the eslint config is JS specific

Reply:
We can add the repsective TS prettier command when we start converting to ts(x) files :+1: